README.md

```diff
@@ -1,4 +1,9 @@
 nfc
 ===
 
-on behalf of W3C NFC working group
\ No newline at end of file
+on behalf of W3C NFC working group
+
+index.html: the current Editor's draft with separate handling of peers and tags.
+
+index2.html: an experimental version which merges peer and tag read handling,
```

Comment:
I would avoid doing this. Just put experimental stuff in the spec and mark it as such. Let's treat the spec as a space for discussion and not yet for implementation.